accepted/2021/experimental-package-shading/experimental-package-shading.md

```diff
@@ -0,0 +1,339 @@
+# Experiment: NuGet Package Shading
+
+**Owner** [Mikayla Hutchinson](https://github.com/mhutch)
+
+## Introduction
+
+Producer-side package shading is an experimental feature that allows a NuGet package author to "shade" a dependency: embed a renamed copy of it in their package. This ensures that consumers of the package get the exact same version that the package author intended, regardless of any other direct or indirect references to that dependency.
+
+**The major downside to producer-side shading** is that the shaded assemblies are included in the app even when they are identical or completely compatible with other shaded or non-shaded copies of the same dependency. This forced redundancy is a problem for scenarios where app size and memory use is highly important such as mobile apps and client-side web apps, **so producer side shading is not intended to ever become non-experimental or recommended for general use**.
+
+The preferred long-term solution is consumer-side shading, where shading is performed holistically for an app, where it shade packages only when necessary, and deduplicate compatible and identical shaded assemblies.
+
+Producer-side shading uses same building blocks as consumer-side shading, but is substantially smaller in scope. It also has the advantage that consumers of a producer-side shading package do not need tooling that understands shading.
```

Comment:
```suggestion
Producer-side shading uses the same building blocks as consumer-side shading, but is substantially smaller in scope. It also has the advantage that people who reference a package with producer-side shading do not need tooling that understands shading.
```

Reply:
I had to read this sentence several times to understand, I think because it contains the word "consumer" when talking about produce-side shading.